indico/modules/events/tracks/templates/management.html

```diff
@@ -1,20 +1,25 @@
 {% extends 'layout/full_content_base.html' %}
+{% from 'events/tracks/_track_list.html' import render_track_list %}
 
 {% block title %}
     {%- trans %}Programme{% endtrans -%}
 {% endblock %}
 
-{% block content %}
-    {% for track in tracks %}
-        <div>
-            {{ track.title }}
-        </div>
-    {% endfor %}
+{% block description %}
+    {%- trans %} Tracks form the programme of the event. Here you can find a list of tracks for this event.
```

Comment:
linebreak after `{%- trans %}`, also use `{%- trans -%}` to avoid surrounding whitespace in the translation string (same at the beginning of `endtrans`)